cloud_governance/policy/ibm/ibm_cost_over_usage.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+import os
+from ast import literal_eval
+
+from cloud_governance.common.clouds.ibm.account.ibm_account import IBMAccount
+from cloud_governance.common.elasticsearch.elastic_upload import ElasticUpload
+from cloud_governance.common.logger.init_logger import logger
+from cloud_governance.common.mails.mail_message import MailMessage
+from cloud_governance.common.mails.postfix import Postfix
+
+
+class IBMCostOverUsage(ElasticUpload):
```

Comment:
Why we are using inheritance and not just a member class for upload ?

Reply:
Previously I need inheritance instead of an object. So when I need I create an inheritance. I will remove it now.